Leanplum-SDK/Classes/MessageTemplates/ViewControllers/LPWebInterstitialViewController.m

```diff
@@ -144,8 +144,16 @@ - (void)addFullscreenConstraints
 - (void)addTemplateConstraints
 {
     [self.webView setTranslatesAutoresizingMaskIntoConstraints:NO];
-    [[self.webView.topAnchor constraintEqualToAnchor:self.view.topAnchor constant:0] setActive:YES];
-    [[self.webView.bottomAnchor constraintEqualToAnchor:self.view.bottomAnchor constant:0] setActive:YES];
+    CGFloat top = 0;
+    CGFloat bottom = 0;
+#if __IPHONE_OS_VERSION_MIN_REQUIRED < __IPHONE_11_0
+    if (@available(iOS 11, *)) {
+#endif
+        top = UIApplication.sharedApplication.delegate.window.safeAreaInsets.top > 20 ? -UIApplication.sharedApplication.delegate.window.safeAreaInsets.top : 0;
```

Comment:
could this just be `-UIApplication.sharedApplication.delegate.window.safeAreaInsets.top` ? Do we actually want to ignore it if it is less than 20?

Reply:
yes, it should work for any height. 
Fixed.